database/buildSchema/07B_template.sql

```diff
@@ -9,6 +9,10 @@ CREATE TABLE public.template (
     id serial PRIMARY KEY,
     name varchar,
     code varchar NOT NULL,
+    serial_pattern jsonb DEFAULT '{
```

Comment:
sorry I missed the discussion mentioned in PR comments, actions are also duplicated when template is duplicated, which seemed to have been the main reason for this ? Any other cases where this can be useful ? Can chat tomorrow